packages/notifier/src/types.js

```diff
@@ -8,6 +8,31 @@
  * @typedef {import('@agoric/promise-kit').PromiseRecord<T>} PromiseRecord
  */
 
+/**
+ * @typedef {*} Passable
+ * A value that can be passed between vats.
+ * TODO This type should be exported by `@agoric/marshal` and imported here.
+ * TODO For these tools to be used for distributed operation, we need a less
+ * precise comparison than `t.is` when testing that these values have arrived.
```

Comment:
Tracking issues?

Reply:
> TODO This type should be exported by `@agoric/marshal` and imported here.
#1958 and https://github.com/Agoric/agoric-sdk/tree/amount-patterns/packages/marshal/src

Reply:
> TODO For these tools to be used for distributed operation, we need a less precise comparison than `t.is` when testing that these values have arrived.

#1959